paymentsheet/src/main/java/com/stripe/android/common/ui/LoadingIndicator.kt

```diff
@@ -9,27 +9,41 @@ import androidx.compose.material.MaterialTheme
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.dimensionResource
 import com.stripe.android.paymentsheet.R
 
 @Composable
-internal fun PaymentSheetLoading(
+internal fun BottomSheetLoadingIndicator(
     modifier: Modifier = Modifier,
 ) {
     val height = dimensionResource(R.dimen.stripe_paymentsheet_loading_container_height)
-    val indicatorSize = dimensionResource(R.dimen.stripe_paymentsheet_loading_indicator_size)
-    val strokeWidth = dimensionResource(R.dimen.stripe_paymentsheet_loading_indicator_stroke_width)
-
     Box(
         contentAlignment = Alignment.Center,
         modifier = modifier
             .fillMaxWidth()
             .height(height),
+    ) {
+        LoadingIndicator()
+    }
+}
```

Comment:
Nit: Pretty sure we can do this via `modifier` now.

```suggestion
    LoadingIndicator(
        modifier = modifier.fillMaxWidth().height(height),
    )
```